src/main/java/mil/dds/anet/threads/AnetEmailWorker.java

```diff
@@ -126,22 +124,34 @@ private void runInternal() {
 		//Send the emails!
 		final List<Integer> processedEmails = new LinkedList<Integer>();
 		for (final AnetEmail email : emails) {
-			if (this.nbOfHoursForStaleEmails != null && email.getCreatedAt().isBefore(Instant.now().atZone(DaoUtils.getDefaultZoneId()).minusHours(nbOfHoursForStaleEmails).toInstant())) {
```

Comment:
You're changing the behaviour for stale emails here. Why?

Reply:
2 changes in behavior: 
1. Previous implementation was not robust: email.getAction().getSubject()) could throw an exception, and that email will never get purged
2. Slight change in behavor as if "email has expired", ANET will try one more time and if that fails then purge it. Just being a bit more conservative - let's say ANET was turned off for a long time and all the emails in the queue have expired.